### PR TITLE
Refactor local account

### PR DESCRIPTION
**Pull request by AbhayAysola**
Refactored `LocalAccount` class. Changed all functions to accept str data typed instead of bytes or nacl types. 

This was done in order to ease the flow between the network and the SDK. Users can use the hex strings returned from the network directly instead of converting to and from byte values or nacl types.

### CODE REVIEW

**Comment by AnonymousDapper** [comment]
The network APIs return strings, yes, but that's because JSON can't encode raw bytes like that. All key responses get converted to their respective objects (namely, VerifyKey) by the validator system. Check out schemas.py if you have questions.

I'll hold off on a review for now so you can read that, so let me know your thoughts when you're done and I'll take another look

**Comment by AbhayAysola** [comment]
> The network APIs return strings, yes, but that's because JSON can't encode raw bytes like that. All key responses get converted to their respective objects (namely, VerifyKey) by the validator system. Check out schemas.py if you have questions.

That's true, but users would still have to import nacl if the keys get converted. 

Honestly, I don't see much point in converting to VerifyKey since we already provide all the required methods in the LocalAccount class. Especially since it takes a few conversions for the VerifyKey object to be in human readable form.

File: aiotnb/core.py [hunk 1]
```diff
@@ -282,20 +281,29 @@ class LocalAccount:
         .. caution:: Do not share this key. This is the private key and can be used to impersonate you.
     """
 
-    def __init__(self, private_key: SigningKey):
+    def __init__(self, signing_key: str):
```
Comment (AnonymousDapper) [inline aiotnb/core.py:284]: The user shouldn't have to manually init the class at all, the `from_key_file` and `generate` classmethods should be the only way of creating a new LocalAccount instance.

**Comment by AnonymousDapper** [comment]
I kinda see where you're coming from, and I'm realizing I may not have explained my idea well enough.

The user does not see the data from the API at all. Everything the user sees has been validated, verified, and parsed into a prettier object that's easier to work with than raw JSON data.

I don't store the Key objects publicly, they do get converted down for readability, but they still exist for ease-of-use. See [here](https://github.com/AnonymousDapper/aiotnb/blob/bank-api/aiotnb/common.py#L79). (I also realized I totally broke this PR by moving the LocalAccount logic out, sorry :( )

Strings are better for readability, but using only strings and encoding/decoding them all the time is a needless waste of resources IMO.

I really appreciate your thoughtfulness, but this doesn't really fit my plan. I can type out a full architecture explanation if it'd help.

File: aiotnb/core.py (hunk 1 above)
Comment (AbhayAysola) [reply aiotnb/core.py:284]: What about when they want to init a class from a hex string?

**Comment by AbhayAysola** [comment]
Okay , @AnonymousDapper I kinda understand your thought process. 

I was thinking we could make another class method called `from_hex_string` or something similar and init a class from a string. 

That could be the way to go. Other than that it seems fine your way.

**Pull request closed without merging**